mobile/src/stateMachine/views/strategies/DidntWorkStrategiesView.tsx

```diff
@@ -0,0 +1,124 @@
+import { ViewState } from '../base';
+import React from 'react';
+import { observer } from 'mobx-react';
+import AppViewModel from 'src/viewModels';
+import { StyleSheet, Text, View, ScrollView, TouchableHighlight, TouchableOpacity, Animated, Dimensions, Alert, SafeAreaView, FlatList } from 'react-native';
+import { MasloPage, Container, Button, BackArrow, GradientChart, Card } from 'src/components';
+import { ScenarioTriggers } from '../../abstractions';
+import TextStyles, { mainFontMedium } from 'src/styles/TextStyles';
+import Colors from '../../../constants/colors/Colors';
+import { StrategyIded } from '../../../../../mobile/src/constants/QoL';
+
+// import { styles } from 'react-native-markdown-renderer';
+
+import AppController from 'src/controllers';
+import { observable } from 'mobx';
+
+const minContentHeight = 300;
+const { width } = Dimensions.get('window');
+
+@observer
+export class DidntWorkStrategiesView extends ViewState {
+
+  private _strategyThatDidntWork: StrategyIded;
+
+    constructor(props) {
+        super(props);
+        this._contentHeight = this.persona.setupContainerHeightForceScrollDown({ transition: { duration: 0} });
+        this.hidePersona();
+        this._strategyThatDidntWork = null; // this.viewModel.strategyThatDidntWork;
+    }
+
+    public get viewModel() {
+        return AppViewModel.Instance.ChooseStrategy;
+    }
+
+    async start() {
+      // this._strategyThatDidntWork = this.viewModel.strategyThatDidntWork;
+      this.forceUpdate();
+    }
+
+    private cancel = () => {
+        this.trigger(ScenarioTriggers.Cancel);
+    }
+
+    onLearnMorePress(id: string) {
+      // MK-TODO: is this a place for learn more
+  }
+
+    onClose = (): void | Promise<void> => this.runLongOperation(async () => {
+        this.showModal({
+            title: `Do you really want to stop? Your progress will not be saved.`,
```

Comment:
This copy is really similar to the one where we say progress WILL be saved. Should we think about emphasizing the 'not' somehow?

Reply:
I think its better to change the copy where it WILL be saved as `Do you really want to stop? Your progress will not be saved.` is used in more places. I will refactor them out into a constants file